.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -1,117 +1,162 @@
 /*
- * Copyright 2007-present the original author or authors.
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I don't think this is accurate. fb-contrib isn't being licensed to the Apache foundation, it's just using their licence boilerplate.

Reply:
Ah, sorry, I misunderstood; this is specifically for the Maven wrapper, not for fb-contrib.